### ISSUE TITLE
[Bug] Extra apostrophe is added after saving

### ISSUE THREAD

**Issue by hockchuanuwc** [theron-wang/VS2022-Editor-Support-for-Tailwind-CSS#42]
**Steps:**
1. Before saving,
```
<button x-on:click="tab = 1" :class="{ 'bg-neutral-500 font-bold': tab === 1, 'bg-neutral-200': tab !== 1 }" class="inline-flex h-12 w-full items-start justify-between rounded p-4">
   Button
</button>
```
2. After saving, an extra apostrophe will be appended before 'bg-neutral-200' css name.
```
<button x-on:click="tab = 1" :class="{ 'bg-neutral-500 font-bold': tab === 1, ''bg-neutral-200': tab !== 1 }" class="inline-flex h-12 w-full items-start justify-between rounded p-4">
   Button
</button>
```

**Extension Version:** 1.3.2
**Visual Studio:** Microsoft Visual Studio Enterprise 2022 (64-Bit) - Current
**Version:** 17.9.7

**Comment by hockchuanuwc** [comment]
The issue was in `.cshtml` file, I didn't test in `.html` file. Yes, the issue is now fixed in the latest 1.4.0 version.

You can close this issue. Thank you for the speedy resolution.

**Comment by theron-wang** [comment]
Great, thanks.
